rust-lib/Cargo.toml

```
@@ -15,9 +15,11 @@ cbindgen = "^0.14"
 openssl = { version = "^0.10", features = ["vendored"] }
 
 [dependencies]
-amplify_derive = "~2.0.6"
+amplify_derive = "~2.2.2"
 log = "0.4"
-rgb_node = "~0.1.1"
+rgb_node = { version = "~0.2.0-beta.1", features = [
+    "server", "client", "embedded", "cli", "shell", "node", "serde",
+    "fungibles", "collectibles", "identities"] }
```

don't need all these features; most of them are internals and are turned on by the main ones.

```suggestion
    "embedded", "shell", "serde",
    "fungibles", "collectibles", "identities"] }
```

---

I thought that enabling all features was temporary, I wasn't planning to leave this change in the PR. Did I understand correctly that `beta.2` will not need any feature in order to build?

---

It will need the features in the list - pls see my code replacement above. And I have released beta-2 yesterday